calico-cloud/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -72,8 +74,23 @@ You must download and set the executable flag each time you get a new version of
 ./tigera-scanner version
 {{imageassuranceversion}}
    ```
+### Integrate the scanner into your build pipeline
+
+You can include the CLI scanner in your CI/CD pipelines (for example, Jenkins, Github actions). Ensure the following:
+
+- The CLI binary is available in the worker or runner where the pipeline is executing
+- If you are running an ephemeral environment in the pipeline, include the download, and update the executable steps in your pipeline to download the scanner on every execution
```

Comment:
Hmm, the way I view this is, regardless of whether their CI platform supports caching of artifacts (which I would imagine all of them do), they need to include an instruction step to download our CLI binary. 

So this bullet point probably just be more just a call out that they need to download the binary onto their CI runner.

Reply:
Changed to: Download the CLI scanner binary onto your CI runner